src/Sulu/Bundle/ContentBundle/Preview/PreviewRenderer.php

```diff
@@ -89,6 +89,8 @@ public function render(PageBridge $content, $partial = false)
         // get controller and invoke action
         $request = new Request($query, $request, [], $cookies);
         $request->attributes->set('_controller', $content->getController());
+        $request->attributes->set('webspace', $content->getWebspaceKey());
+        $request->attributes->set('locale', $content->getLanguageCode());
```

Comment:
Ok, I see now what's going on here... In the admin we are usually setting these variables via the `query` and not via the `attributes` object. But in the `AdminRequestProcessor` we are using the `get` method, which checks all the `ParameterBag`s (btw I read somewhere that this method should only be used when really necessary). Could you maybe change the `attributes` here to `query`, so that it is less confusing for me? 😃